src/cli/argument-parser.ts

```diff
@@ -387,8 +386,8 @@ export default class CLIArgumentParser {
     }
 
     private async _parseDashboardOptions (): Promise<void> {
-        if (this.opts.dashboard)
-            this.opts.dashboard = await getDashboardOptions(this.opts.dashboard as string);
+        if (this.opts.dashboardOptions)
+            this.opts.dashboardOptions = await getDashboardOptions(this.opts.dashboardOptions as string);
```

Comment:
https://github.com/DevExpress/testcafe/blob/master/src/cli/cli.js#L120

The object names are not match (opts.dashboardOptions here and opts.dashboard in the file above). The CLI does not work at this moment.

How to check locally:

1. Update the testcafe-reporter-dashboard version to 0.2.6-rc.1
2. Build and pack testcafe into tarball
3. Install testcafe from the tarball to some test project
4. Run test from runner. It works:

```
const createTestCafe        = require('testcafe');
let runner                  = null;

createTestCafe('localhost', 9222, 9223)
    .then(testcafe => {
        runner = testcafe.createRunner();
    })
    .then(() => {
        return runner
            .src('test.js')
            .browsers(['chrome'])
            .dashboard({
                token: 'some-token'
            })
            .run();
    });
```
5. Run from CLI. The default spec reporter will be used:

`npx testcafe chrome test.js --dashboard-options token=some-token`